webauthn4j-core/src/main/java/com/webauthn4j/validator/attestation/statement/tpm/TPMAttestationStatementValidator.java

```diff
@@ -127,7 +127,7 @@
         algJcaName = getAlgJcaName(hashAlg);
 
         byte[] pubAreaDigest = MessageDigestUtil.createMessageDigest(algJcaName).digest(pubArea.getBytes());
-        if (!Arrays.equals(pubAreaDigest, certifyInfo.getName().getDigest())) {
+        if (!MessageDigest.isEqual(pubAreaDigest, certifyInfo.getName().getDigest())) {
```

Comment:
`pubAreaDigest` is also known data to client side(potential attacker) because it is part of a message, there is no risk of timing attack too.